templates/site/js/node.js

```diff
@@ -66,8 +66,7 @@ const converters = {
                     name = service.name;
                 }
 
-                if (link.rel === "service-desc" ) {
-                    name="";
+                if (service.description != "" ) {
```

Comment:
Description is a required field (https://github.com/DACCS-Climate/Marble-node-registry/blob/main/node_registry.schema.json#L85) so you don't need to check for its existence.

Reply:
Got rid of if-statement and moved description value assignment up to where description variable is created.